1365.py

```diff
@@ -0,0 +1,16 @@
+class Solution:
+    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
+        sorted_nums = sorted(nums)
+        ans = []
+        
+        for num in nums:
+            count = 0
+            
+            for sorted_num in sorted_nums:
```

Comment:
`x = sorted_nums.index(num)` does the same thing as your inner `for` loop. However, both the `index()` function and your `for` loop take linear time to complete this task for every `num` in `nums`, making the overall time complexity `O(N^2)`. 

Since `sorted_nums` is already in a sorted order, can you think of a better way to locate the `x` mentioned above?